WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -343,6 +346,32 @@ + (NSString *)dismissAlertButtonSelector
   return FBDismissAlertButtonSelector;
 }
 
+#if !TARGET_OS_TV
++ (void)setScreenshotOrientation:(NSString *)orientation
+{
+  // Only UIInterfaceOrientationUnknown is over iOS 8. Others are over iOS 2.
+  // https://developer.apple.com/documentation/uikit/uiinterfaceorientation/uiinterfaceorientationunknown
+  if (orientation == nil) {
+    FBScreenshotOrientation = UIInterfaceOrientationUnknown;
+  } else if ([orientation.lowercaseString isEqualToString:@"portrait"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationPortrait;
+  } else if ([orientation.lowercaseString isEqualToString:@"portraitupsidedown"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationPortraitUpsideDown;
+  } else if ([orientation.lowercaseString isEqualToString:@"landscaperight"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationLandscapeRight;
+  }else if ([orientation.lowercaseString isEqualToString:@"landscapeleft"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationLandscapeLeft;
+  } else {
+    FBScreenshotOrientation = UIInterfaceOrientationUnknown;
+  }
+}
+
++ (NSInteger)screenshotOrientation
+{
+  return FBScreenshotOrientation;
```

Comment:
this might be confusing that we set a string, but return an integer. I would rather return the same value that has been set except for `auto` for `UIInterfaceOrientationUnknown `